models/engine/file_storage.py

```diff
@@ -70,4 +70,20 @@ def close(self):
         self.reload()
 
     def get(self, cls, id):
-        """Returns the object based on the class name and its ID"""
+        """ method to retrieve one object """
+        if cls:
+            for key, value in self.__objects.items():
```

Comment:
I don't see the use of the key parameter in this function.  I think is better delete that